### ISSUE TITLE
alias_method_chain deprecation warnings  

### ISSUE THREAD

**Issue by cjjuice** [PavelTyk/sendgrid-rails#18]
The use of alias_method_chain in

https://github.com/PavelTyk/sendgrid-rails/blob/77957e637d2badff9e4981caadfe76aefd7adf83/lib/send_grid.rb#L11

is throwing a deprecation warning in rails 5, Module#prepend is what it says to use. I believe this is only available > ruby 2.0.0

Thoughts? Would something like this work?   

```ruby
  module InstanceMethods
    prepend SendGrid

    def mail(headers={}, &block)
      @sendgrid_header = SendGrid::ApiHeader.new
      super
    end

    def open_tracking(enabled = true)
      add_filter_setting(:opentrack, :enabled, enabled ? 1 : 0) unless enabled.nil?
    end
  end
```

**Comment by PavelTyk** [comment]
That makes sense to me. Can you please do a pull request? Also make sure tests do not fail.

Thanks for pointing that out.

**Comment by PavelTyk** [comment]
Solved with #19 
Thanks @cjjuice
